capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -27,9 +27,18 @@
   private static DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
   /**
-   * Will post a new restaurant into datastore.
-   * @param request sent by the frontend.
-   * @param response to the frontend.
+   * Will put the information gathered from the signup page and add the
+   * information into their respective properties for .
+   * @param request sent by the frontend. Requires the user to be logged in
+   *                and have the following parameters in the request body:
+   *                "name": name of the restaurant,
+   *                "cuisine": list of cuisines of the restaurant as CSV,
+   *                "story": story of the restaurant,
+   *                "phone": cellphone of the restaurant,
+   *                "website": link to their website,
+   * @param response will redirect to the home page of the website. If the
+   *                 user is not logged in when requesting the POST, the
+   *                 response will have a SC_FORBIDDEN status.
```

Comment:
Awesome job :D I could easily write the front-end code to use this call after reading this!